### ISSUE TITLE
`@yamada-ui/charts` warning

### ISSUE THREAD

**Issue by 108yen** [yamada-ui/yamada-ui#1443]
### Which package has an issue?

@yamada-ui/charts

### Package Version

1.3.9

### Description

Warnings in `XAxis`, `YAxis` and `ReferenceLine`.
```
Warning: XAxis: Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.
    at XAxis2 (http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/recharts.js?v=22736b93:25401:22)
    at svg
    at Surface (http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/recharts.js?v=22736b93:1907:25)
    at div
    at ChartLayoutContextProvider2 (http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/recharts.js?v=22736b93:21888:28)
    at CategoricalChartWrapper (http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/recharts.js?v=22736b93:26712:7)
    at div
    at http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/recharts.js?v=22736b93:5245:21
    at div
    at http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/chunk-UJG22J73.js?v=22736b93:148:45
    at http://localhost:6006/packages/core/src/styled.ts?t=1715057196935:38:35
    at http://localhost:6006/packages/components/charts/src/line-chart.tsx?t=1715060374113:33:33
    at section
    at http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/chunk-UJG22J73.js?v=22736b93:148:45
    at http://localhost:6006/packages/core/src/styled.ts?t=1715057196935:38:35
    at http://localhost:6006/packages/components/layouts/src/container.tsx?t=1715057196935:11:33
    at App (http://localhost:6006/.storybook/components.tsx?t=1715057702863:24:16)
    at LoadingProvider (http://localhost:6006/packages/components/loading/src/loading-provider.tsx?t=1715057196935:42:3)
    at MotionConfig (http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/framer-motion.js?v=22736b93:6985:25)
    at EnvironmentProvider (http://localhost:6006/packages/providers/src/environment-provider.tsx?t=1715057196935:10:3)
    at ColorModeProvider (http://localhost:6006/packages/core/src/providers/color-mode-provider.tsx?t=1715057196935:17:14)
    at ThemeProvider2 (http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/chunk-UJG22J73.js?v=22736b93:195:21)
    at ThemeProvider (http://localhost:6006/packages/core/src/providers/theme-provider.tsx?t=1715057196935:20:10)
    at UIProvider (http://localhost:6006/packages/providers/src/ui-provider.tsx?t=1715057196935:14:3)
    at StoryProvider (http://localhost:6006/.storybook/components.tsx?t=1715057702863:13:33)
    at unboundStoryFn (http://localhost:6006/sb-preview/runtime.js:111:5353)
    at ErrorBoundary (http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/chunk-W6OAWYBP.js?v=22736b93:32:5)
    at WithCallback (http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/chunk-AYW4C4W3.js?v=22736b93:15:23)
```

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

https://github.com/recharts/recharts/issues/3615

**Comment by 108yen** [comment]
Will be fixed at `recharts@v2.13.0`

**Comment by hirotomoyamada** [comment]
Fixed #3101
